Google.PowerShell/Compute/GceHealthCheckCmdlets.cs

```diff
@@ -0,0 +1,545 @@
+﻿using Google.Apis.Compute.v1;
+using Google.Apis.Compute.v1.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Google.PowerShell.ComputeEngine
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Adds a Google Compute Engine health check.
+    /// </para>
+    /// <para type="descritpion">
+    /// Adds a Google Compute Engine health check. Use this cmdlet to create both http and https health checks.
```

Comment:
HTTP and HTTPS. Also, is there a link to the docs you can point to? i.e. something that describes what health checks are used for with examples?